hikari/impl/rest.py

```diff
@@ -768,6 +768,7 @@ async def edit_channel(
         topic: undefined.UndefinedOr[str] = undefined.UNDEFINED,
         nsfw: undefined.UndefinedOr[bool] = undefined.UNDEFINED,
         bitrate: undefined.UndefinedOr[int] = undefined.UNDEFINED,
+        video_quality_mode: undefined.UndefinedOr[typing.Union[channels_.VideoQualityMode, int]] = undefined.UNDEFINED,
```

Comment:
This arg should also be added to the guild builder and create_guild_voice_channel method